operators/src/engine/operator.rs

```diff
@@ -175,6 +175,16 @@ pub trait InitializedVectorOperator: Send + Sync {
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct CanonicOperatorName(serde_json::Value);
 
+impl CanonicOperatorName {
+    pub fn new(value: serde_json::Value) -> Self {
+        CanonicOperatorName(value)
+    }
+
+    pub fn byte_size(&self) -> usize {
+        std::mem::size_of_val(self)
+    }
```

Comment:
maybe we should check that this works?

Reply:
actually: id didn't. And moreover it appears that json::Value can be vastly bigger in memory than a json string. So I changed it to use the byte representation like you suggested

06ca2acb65400e096f0091afc467fccf0bcf5633